datasets/tweet_eval/README.md

```diff
@@ -0,0 +1,452 @@
+
+    `1`: neutral
+
+    `2`: positive
+
+For `stance_abortion` config:
+
+- `text`: a `string` feature containing the tweet.
+
+- `label`: an `int` classification label with the following mapping:
+
+    `0`: none
+
+    `1`: against
+
+    `2`: favor
+
+For `stance_atheism` config:
+
+- `text`: a `string` feature containing the tweet.
+
+- `label`: an `int` classification label with the following mapping:
+
+    `0`: none
+
+    `1`: against
+
+    `2`: favor
+
+For `stance_climate` config:
+
+- `text`: a `string` feature containing the tweet.
+
+- `label`: an `int` classification label with the following mapping:
+
+    `0`: none
+
+    `1`: against
+
+    `2`: favor
+
+For `stance_feminist` config:
+
+- `text`: a `string` feature containing the tweet.
+
+- `label`: an `int` classification label with the following mapping:
+
+    `0`: none
+
+    `1`: against
+
+    `2`: favor
+
+For `stance_hillary` config:
+
+- `text`: a `string` feature containing the tweet.
+
+- `label`: an `int` classification label with the following mapping:
+
+    `0`: none
+
+    `1`: against
+
+    `2`: favor
+
+
+
+### Data Splits
+
+| name            | train | validation | test  |
+| --------------- | ----- | ---------- | ----- |
+| emoji           | 45000 | 5000       | 50000 |
+| emotion         | 3257  | 374        | 1421  |
+| hate            | 9000  | 1000       | 2970  |
+| irony           | 2862  | 955        | 784   |
+| offensive       | 11916 | 1324       | 860   |
+| sentiment       | 45615 | 2000       | 12284 |
+| stance_abortion | 587   | 66         | 280   |
+| stance_atheism  | 461   | 52         | 220   |
+| stance_climate  | 355   | 40         | 169   |
+| stance_feminist | 597   | 67         | 285   |
+| stance_hillary  | 620   | 69         | 295   |
+
+## Dataset Creation
+
+### Curation Rationale
+
+[Needs More Information]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[Needs More Information]
+
+#### Who are the source language producers?
+
+[Needs More Information]
+
+### Annotations
+
+#### Annotation process
+
+[Needs More Information]
+
+#### Who are the annotators?
+
+[Needs More Information]
+
+### Personal and Sensitive Information
+
+[Needs More Information]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[Needs More Information]
+
+### Discussion of Biases
+
+[Needs More Information]
+
+### Other Known Limitations
+
+[Needs More Information]
+
+## Additional Information
+
+### Dataset Curators
+
+Francesco Barbieri, Jose Camacho-Collados, Luis Espiinosa-Anke and Leonardo Neves through Cardiff NLP.
+
+### Licensing Information
+
+[Needs More Information]
+### Citation Information
+
+@inproceedings{barbieri2020tweeteval,
+title={{TweetEval:Unified Benchmark and Comparative Evaluation for Tweet Classification}},
+author={Barbieri, Francesco and Camacho-Collados, Jose and Espinosa-Anke, Luis and Neves, Leonardo},
+booktitle={Proceedings of Findings of EMNLP},
+year={2020}
+}
```

Comment:
````suggestion
```
@inproceedings{barbieri2020tweeteval,
title={{TweetEval:Unified Benchmark and Comparative Evaluation for Tweet Classification}},
author={Barbieri, Francesco and Camacho-Collados, Jose and Espinosa-Anke, Luis and Neves, Leonardo},
booktitle={Proceedings of Findings of EMNLP},
year={2020}
}
```
````